.travis.yml

```diff
@@ -1,7 +1,50 @@
-sudo: false
-language: javascript
-
-install: npm install
-
-
-script: npm run test:unit:js && npm run test:smoke:typescript
+jobs:
+  include:
+    - stage: test
+      sudo: false
+      language: javascript
+      install: npm install
+      script: npm run test:unit:js && npm run test:smoke:typescript
+    - stage: build android
+      language: android
+      sudo: false
+      android:
+        components:
+         - tools
+         - platform-tools
+         - tools
```

Comment:
Is tools being listed twice intentional? There used to be a Travis bug that required a similar workaround, so this can be ignored if it's still an issue

Reply:
I think this was necessary, although I can try without to double check